src/test/scala/cognite/spark/v1/FlexibleDataModelsRelationTest.scala

```diff
@@ -362,20 +366,36 @@ class FlexibleDataModelsRelationTest extends FlatSpec with Matchers with SparkTe
       space = spaceExternalId,
       externalId = endNodeExtId
     )
+    val directNodeReference = DirectRelationReference(
+      space = spaceExternalId,
+      externalId = startNodeExtId
+    )
 
-    val (viewAll, viewNodes, viewEdges, allInstanceExternalIds) = (for {
+    val (_, _, viewEdges, allInstanceExternalIds) = (for {
       (viewAll, viewNodes, viewEdges) <- setupAllListAndNonListPropertyTest
-      nodeIds <- createTestInstancesForView(viewNodes, None, None)
-      edgeIds <- createTestInstancesForView(viewEdges, Some(startNodeRef), Some(endNodeRef))
-      allIds <- createTestInstancesForView(viewAll, Some(startNodeRef), Some(endNodeRef))
+      nodeIds <- createTestInstancesForView(
+        viewNodes,
+        directNodeReference = directNodeReference,
+        None,
+        None)
+      edgeIds <- createTestInstancesForView(
+        viewEdges,
+        directNodeReference,
+        Some(startNodeRef),
+        Some(endNodeRef))
+      allIds <- createTestInstancesForView(
+        viewAll,
+        directNodeReference,
+        Some(startNodeRef),
+        Some(endNodeRef))
     } yield (viewAll, viewNodes, viewEdges, nodeIds ++ edgeIds ++ allIds)).unsafeRunSync()
 
-    val readNodesDf = readRows(
-      viewSpaceExternalId = spaceExternalId,
-      viewExternalId = viewNodes.externalId,
-      viewVersion = viewNodes.version,
-      instanceSpaceExternalId = spaceExternalId
-    )
+//    val readNodesDf = readRows(
+//      viewSpaceExternalId = spaceExternalId,
+//      viewExternalId = viewNodes.externalId,
+//      viewVersion = viewNodes.version,
+//      instanceSpaceExternalId = spaceExternalId
+//    )
```

Comment:
should this be removed? or it's gonna be used later?

Reply:
same for the blocks right bellow this one

Reply:
as mentioned in the description
```
 some tests are ignored due to pending fdm bug fixes.
```